internal/cmd/test.go

```diff
@@ -74,12 +82,19 @@ func CMDServerTestingMain(suite *testing.M) {
 	exit.Exit(code)
 }
 
-func RunCMDTest(test *testing.T, mainFunc func(), args []string, logLevel log.LogLevel) (string, string, int, error) {
+func RunCMDTest(test *testing.T, mainFunc func(), args []string, logLevel log.LogLevel, removeTempDirsAfterEachTest bool) (string, string, int, error) {
```

Comment:
The name "removeTempDirsAfterEachTest" is not good.
This function runs one test, but the name implies multiple tests.